### DISCUSSION TITLE
Henri pis sa gang

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#31] [Q&A]
Commentaires sur ce quiz.

**Accès**
- Serveur de développement (protégé) :  https://github.com/Jpcote14/Gr2_CoteJp_timQuiz_complet
- Dépôt Github (privé pour le moment): https://github.com/Jpcote14/Gr2_CoteJp_timQuiz_complet
- Github.io :

**Comment by alexandradias** [comment]
Pour commencer, un grand merci à toi de m'avoir fait découvrir ce sujet que je ne connais pas :)

J'aime beaucoup comment tu as géré l'ensemble de ton projet en y incluant une image de fond afin de transmettre une continuité à ton quiz.
J'ai l'impression qu'il y a des chemins d'image qui sont rompus, les images y sont manquantes, dans le footer et 

J'ai l'impression que les réponses ne correspondent pas toujours, sauf erreur de ma part, sur la question 3 j'ai cocher Texas et ma réponse a été indiqué comme fausse malgré la rétroaction qui semble me donner le même choix de réponse.
Je te glisse des captures pour que tu comprennes mieux mes écrits ;).

J'aurais peut être centré la partie du score en version table pour une meilleure harmonie.
Par contre génial d'avoir intégré la possibilité de recommencer le quiz.

Au niveau de ton html, le validateur m'indique certaines erreurs encore présentes, j'imagine que tu as peut être manqué de temps pour pouvoir peaufiner le balisage :/
En désactivant le javascript, je vois parfaitement tes questions, mais ton intro est coupée, afin d'y remédier j'aurais sur accueil-droite (width:100%) avec tous les autres enfants avec une width:75%, text-align:center. Je te glisse une capture pour mieux comprendre ;).
Tu as géré le responsive mobile et table comme demandé.

Encore bravo pour ton travail!


<img width="1440" alt="Capture d’écran 2023-05-10 à 14 41 25" src="https://github.com/integration2/quizTIM2023/assets/52402064/95cab11e-cfb8-4257-88eb-c791eda5bd7b">
<img width="1440" alt="Capture d’écran 2023-05-10 à 14 44 53" src="https://github.com/integration2/quizTIM2023/assets/52402064/1262452c-c445-4985-82bd-812d959c8eac">
<img width="1440" alt="Capture d’écran 2023-05-10 à 14 53 16" src="https://github.com/integration2/quizTIM2023/assets/52402064/f510f292-155f-44ba-bb01-aaf0956acff1">
